src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java

```diff
@@ -183,6 +186,8 @@
     private EditorMouseListener mouseListener;
     private EditorMouseMotionListener mouseMotionListener;
 
+    public List<String> completionTriggers;
+    public List<String> signatureTriggers;
```

Comment:
should it be List<Character> ?

Reply:
lsp4j has defined both completion-triggers and signature-triggers as `List<String>` (Not sure the reason why) so I preserved the type as it is.